README.md

```diff
@@ -228,6 +228,14 @@ superface.run(
 );
 ```
 
+## Known Limitations
+
+### Multiple matching requests for the same use case
+
+Recordings make it possible to run tests without calling the live API. This works by trying to match a request to the requests in the existing recordings. If a match is found, the recorded response is returned. However, since the testing client saves recording for use-cases in a single file, it means multiple matching requests for the same use-case will overwrite each other.
```

Comment:
Just a few suggestions, behind the cause of rewriting recordings are same inputs inside identical use-cases, recording file is created or loaded in each call of `run()` method.

```suggestion
Recordings make it possible to run tests without calling the live API. This works by trying to match a request to the requests in the existing recordings. If a match is found, the recorded response is returned. However, since the testing client saves recording for each test run in a single file, it means multiple matching requests for the same use-case and input will overwrite each other.
```